course6/toyapp_thomasdhuys/drawlib/projector.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "index.h"
+#include "rgb.h"
+
+namespace Draw {
+
+struct Index2D
```

Comment:
And this, i think, would be more descriptive as "Pixel"

Reply:
renamed